src/FSharp.Analyzers/FSharp.Analyzers.fsproj

```diff
@@ -1,34 +1,36 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
-        <GenerateDocumentationFile>true</GenerateDocumentationFile>
-        <IsPackable>true</IsPackable>
-        <AssemblyName>G-Research.FSharp.Analyzers</AssemblyName>
-        <Tailcalls>true</Tailcalls>
-        <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
-        <DevelopmentDependency>true</DevelopmentDependency>
-        <NoPackageAnalysis>true</NoPackageAnalysis>
-        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_AddAnalyzersToOutput</TargetsForTfmSpecificContentInPackage>
-    </PropertyGroup>
+  <PropertyGroup>
```

Comment:
We should lock the `indent_size` in our `.editorconfig` for this.

Reply:
Oh sorry, that was ugly.